api/XMLHttpRequest.json

```diff
@@ -0,0 +1,1736 @@
+                "version_added": null
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "document": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/XMLHttpRequest/responseType",
+            "support": {
+              "webview_android": {
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": "18"
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "11"
+              },
+              "firefox_android": {
+                "version_added": null
+              },
+              "ie": {
+                "notes": "<code>json</code> as value is not supported.",
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "6.1"
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "json": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/XMLHttpRequest/responseType",
+            "support": {
+              "webview_android": {
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": "31"
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "10"
+              },
+              "firefox_android": {
+                "version_added": null
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": [
+                {
+                  "version_added": "12",
+                  "version_removed": "16"
+                },
+                {
+                  "version_added": "17"
```

Comment:
Swap this around, first the object with version_added: 17 after that the one with 12-16. (anti-chronological)

Reply:
The removal is suspicious to me, but I can't disprove it. It's still in Chrome. Since Opera is chromium based, I don't know how keep this out. Seems like they'd need to reconcile a merge conflict with every version of Chrome.